c_binding/utils.c

```diff
@@ -3,7 +3,7 @@
  * This library is free software; you can redistribute it and/or
  * modify it under the terms of the GNU Lesser General Public
  * License as published by the Free Software Foundation; either
- * version 3 of the License, or (at your option) any later version.
+ * version 2.1 of the License, or (at your option) any later version.
```

Comment:
For any of the files that had previously been changed to LGPL v3, I'd rather see a separate PR to fix that issue. And we should get that committed ASAP. So, just cherry-pick this patch out of this PR and create a new one.